src/css/components/_adaptive.scss

```diff
@@ -1,44 +1,70 @@
-// When the player is absurdly tiny, display nothing but:
+// When the player is "medium" and higher, display everything by default.
+//
+// When the player is "small", display only:
 // - Play button
+// - Volume Mute button
+// - Progress bar
+// - Subs-Caps button
+// - Fullscreen button
+//
+// When the player is "small", display only:
+// - Play button
+// - Volume Mute button
+// - Progress bar
+// - Fullscreen button
+//
+// When the player is "tiny", display only:
+// - Play button
+// - Volume Mute button
 // - Fullscreen Button
-.video-js.vjs-layout-tiny:not(.vjs-fullscreen) {
-  .vjs-custom-control-spacer {
-    @include flex(auto);
-    display: block;
-  }
+//
+.video-js:not(.vjs-fullscreen) {
 
-  &.vjs-no-flex .vjs-custom-control-spacer { width: auto; }
+  &.vjs-layout-small,
+  &.vjs-layout-x-small,
+  &.vjs-layout-tiny {
+    .vjs-current-time,
+    .vjs-time-divider,
+    .vjs-duration,
+    .vjs-remaining-time,
+    .vjs-playback-rate,
+    .vjs-chapters-button,
+    .vjs-descriptions-button,
+    .vjs-captions-button,
+    .vjs-subtitles-button,
+    .vjs-audio-button,
+    .vjs-volume-control {
+      display: none;
+    }
 
-  .vjs-current-time, .vjs-time-divider, .vjs-duration, .vjs-remaining-time,
-  .vjs-playback-rate, .vjs-progress-control,
-  .vjs-mute-control, .vjs-volume-control, .vjs-volume-panel,
```

Comment:
Removed `.vjs-mute-control` and `.vjs-volume-panel` from the hidden elements lists.